app/src/components/views/shop.jsx

```diff
@@ -1,19 +1,12 @@
-import React from 'react';
-import { render } from 'react-dom';
-import { Navbar, NavItem, Icon } from 'react-materialize';
-
-import { Row, Col, Card, CardTitle, Button } from 'react-materialize';
-import { Tabs, Tab } from 'components/Tabs';
 import { LOGOUT_TIMER } from 'common/constants';
-
-import { serviceManager } from 'services';
-
-import { ClickProxy, CheckoutModal } from 'components/modals.jsx';
-
-import { Subject, Observable } from 'rxjs';
-
 import { CatalogItem } from 'components/CatalogItem.jsx';
+import { CheckoutModal, ClickProxy } from 'components/modals.jsx';
 import { StackItem } from 'components/StackItem.jsx';
+import { Tab, Tabs } from 'components/Tabs';
+import React from 'react';
+import { Button, Col, Row } from 'react-materialize';
+import { Observable, Subject } from 'rxjs';
+import { serviceManager } from 'services';
```

Comment:
Not sure I'm fan of a pure alphabetical sort of imports, I like the way isort does it:

One block for system-wide/global dependencies (like React and rx), then a block for internal references to other components etc. and finally local dependencies like child components or styling.

It's really nitpicky though.

Reply:
Same here, I just followed eslint, and had a plugin that automatically sorted them.